[2025-01-31T04:56:07.408+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1_v2.second_task scheduled__2025-01-03T00:00:00+00:00 [queued]>
[2025-01-31T04:56:07.437+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1_v2.second_task scheduled__2025-01-03T00:00:00+00:00 [queued]>
[2025-01-31T04:56:07.438+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-01-31T04:56:07.693+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): second_task> on 2025-01-03 00:00:00+00:00
[2025-01-31T04:56:07.736+0000] {standard_task_runner.py:57} INFO - Started process 9663 to run task
[2025-01-31T04:56:07.747+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag1_v2', 'second_task', 'scheduled__2025-01-03T00:00:00+00:00', '--job-id', '2914', '--raw', '--subdir', 'DAGS_FOLDER/dag_example.py', '--cfg-path', '/tmp/tmpd39p6x5k']
[2025-01-31T04:56:07.749+0000] {standard_task_runner.py:85} INFO - Job 2914: Subtask second_task
[2025-01-31T04:56:11.946+0000] {task_command.py:415} INFO - Running <TaskInstance: dag1_v2.second_task scheduled__2025-01-03T00:00:00+00:00 [running]> on host f8bdfcb8bca5
[2025-01-31T04:56:13.591+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='anhtupham99' AIRFLOW_CTX_DAG_ID='dag1_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-03T00:00:00+00:00'
[2025-01-31T04:56:13.593+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-01-31T04:56:13.596+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo I am the second task, I run after the first task']
[2025-01-31T04:56:13.698+0000] {subprocess.py:86} INFO - Output:
[2025-01-31T04:56:13.699+0000] {subprocess.py:93} INFO - I am the second task, I run after the first task
[2025-01-31T04:56:13.700+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-01-31T04:56:13.839+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1_v2, task_id=second_task, execution_date=20250103T000000, start_date=20250131T045607, end_date=20250131T045613
[2025-01-31T04:56:13.932+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-01-31T04:56:14.006+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
